src/jm_view_server/app.py

```
@@ -835,7 +835,7 @@ def open_directory(self, directory):
                     subprocess.Popen(['open', path])          # 直接进入该目录
             elif sys.platform.startswith('win'):
                 if reveal or not is_dir:
-                    subprocess.Popen(f'explorer /select,"{path}"')  # 选中
+                    subprocess.Popen(['explorer', f'/select,{path}'])  # 选中
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)